### ISSUE TITLE
[RFC] Add empty state to changelog

### ISSUE THREAD

**Issue by anketwokings** [bolt/bolt#6623]
I recently wanted to check on a live website what the client did wrong. You know, they file a bug request 'it's broken', and I wanted to check if they themselves indeed unpublished a page.

## Problem
* Checked changelog -> empty
* First thought: "must be broken, because empty"
* Expected: A list of changes by the editor / a history of changed pages, just like the small one on the dashboard.

Asked around, and apparently, bc technical and server issues, empty is the default, unless you turn it on ( maybe for a short time only).

## Solution
I suggest a nice self-explaining empty state, to explain
* how to turn it on
* the pros and cons of turning it on, and maybe a link to bolt docs if the explanation would be too long.

Could be easily implemented by adding a

```
<div class="alert alert-info alert-dismissible">
    <button type="button" class="close" data-dismiss="alert">×</button>
    Your text here 
</div>
```

## Example 

![example_emptystate_changelog](https://cloud.githubusercontent.com/assets/5583671/25741254/2f78541e-318a-11e7-94d7-5f5372303994.jpg)

**Comment by GawainLynch** [comment]
Totally has my vote :+1:

**Comment by zomars** [comment]
This makes sense, it took me a while to notice that it was disabled by default.

**Comment by CarsonF** [comment]
If it's not enabled we shouldn't even show the table or trim/clear buttons.

**Comment by GawainLynch** [comment]
> If it's not enabled we shouldn't even show the table or trim/clear buttons.

:+1: 

> Maybe not even show the link in the menu?

But it could have previous records changes saved … but yeah, "balance"

**Comment by anketwokings** [comment]
> If it's not enabled we shouldn't even show the table or trim/clear buttons.
Totally makes sense

> Maybe not even show the link in the menu?
Kind of depends; if you want people to know without too much searching i'd let the link stay, if you want to show it only for people who know where to find it you could hide it.

I'd personally be for keeping the link and adding a good empty state, to keep it more user-friendly.

**Comment by bobdenotter** [comment]
Fixed in #6653
